usr/share/rear/lib/borg-functions.sh

```diff
@@ -91,6 +91,11 @@ function borg_set_vars {
 
     # Set file to save borg stderr output
     BORGBACKUP_STDERR_FILE=$TMP_DIR/borg_stderr
+
+    BORGBACKUP_RESTORE_ARCHIVES_SHOW_NUMBER=${BORGBACKUP_RESTORE_ARCHIVES_SHOW_NUMBER:-10}
+
+    [[ $BORGBACKUP_RESTORE_ARCHIVES_SHOW_NUMBER -ge 0 ]] \
+        || Error "BORGBACKUP_RESTORE_ARCHIVES_SHOW_NUMBER is $BORGBACKUP_RESTORE_ARCHIVES_SHOW_NUMBER, but it must be >= 0!"
```

Comment:
If you like you could simplify the error message to
```
Error "BORGBACKUP_RESTORE_ARCHIVES_SHOW_NUMBER '$BORGBACKUP_RESTORE_ARCHIVES_SHOW_NUMBER' must be >= 0"
```

Please at least leave out the exclamation mark at the end
because shouting at the poor user who made a mistake
won't actually help to make our planet Earth a better place ;-)

FYI:
Usually I prefer to show variable values within `" ... '$var' ... "`
so also empty or blank values become visible as `... '' ...` or `... ' ' ...`

Reply:
Thanks for that hint.

See commit 6cc4b948 `Don't shout at poor user, who made a mistake ;-)` for it.